HPXMLtoOpenStudio/resources/defaults.rb

```diff
@@ -5591,6 +5821,317 @@ def self.get_ceiling_fan_months(weather)
+        next if !system_ids.include?(clothes_dryer.id)
+        next if clothes_dryer.fuel_type != HPXML::FuelTypeElectricity
+
+        if clothes_dryer.is_vented
+          watts += 5760 # FIXME: not 2640?
+          breaker_spaces += 1
+        else # HP
+          if voltage == 120
+            watts += 996
+          else
+            watts += 860
+            breaker_spaces += 1
+          end
+        end
+        breaker_spaces += 1
+      end
+    elsif type == HPXML::ElectricPanelLoadTypeDishwasher
+      hpxml_bldg.dishwashers.each do |dishwasher|
+        next if !system_ids.include?(dishwasher.id)
+
+        watts += 1200
+        breaker_spaces += 1
+      end
+    elsif type == HPXML::ElectricPanelLoadTypeRangeOven
+      hpxml_bldg.cooking_ranges.each do |cooking_range|
+        next if !system_ids.include?(cooking_range.id)
+        next if cooking_range.fuel_type != HPXML::FuelTypeElectricity
+
+        if voltage == 120
+          watts += 1800
+        else
+          watts += 12000
+          breaker_spaces += 1
+        end
+        breaker_spaces += 1
+      end
+    elsif type == HPXML::ElectricPanelLoadTypeMechVent
+      hpxml_bldg.ventilation_fans.each do |ventilation_fan|
+        next if !system_ids.include?(ventilation_fan.id)
+        next if ventilation_fan.is_shared_system
+
+        if [HPXML::LocationKitchen, HPXML::LocationBath].include?(ventilation_fan.fan_location)
+          watts += ventilation_fan.count * ventilation_fan.fan_power
+        elsif not ventilation_fan.fan_power.nil?
+          watts += ventilation_fan.fan_power
+        else
+          watts += 3000 # FIXME: base-mechvent-cfis-no-additional-runtime.xml, e.g., has no FanPower defaulted
+        end
+      end
+      breaker_spaces += 1 # intentionally outside the ventilation_fans loop
+    elsif type == HPXML::ElectricPanelLoadTypePermanentSpaHeater
+      hpxml_bldg.permanent_spas.each do |permanent_spa|
+        next if !system_ids.include?(permanent_spa.heater_id)
+        next if ![HPXML::HeaterTypeElectricResistance, HPXML::HeaterTypeHeatPump].include?(permanent_spa.heater_type)
+
+        watts += 1000 # FIXME
+        breaker_spaces += 2
+      end
+    elsif type == HPXML::ElectricPanelLoadTypePermanentSpaPump
+      hpxml_bldg.permanent_spas.each do |permanent_spa|
+        next if !system_ids.include?(permanent_spa.pump_id)
+
+        watts += 1491
+        breaker_spaces += 1
+      end
+    elsif type == HPXML::ElectricPanelLoadTypePoolHeater
+      hpxml_bldg.pools.each do |pool|
+        next if !system_ids.include?(pool.heater_id)
+        next if ![HPXML::HeaterTypeElectricResistance, HPXML::HeaterTypeHeatPump].include?(pool.heater_type) # FIXME: probably want to separate out HeatPump here
+
+        watts += 27000
+        breaker_spaces += 2
+      end
+    elsif type == HPXML::ElectricPanelLoadTypePoolPump
+      hpxml_bldg.pools.each do |pool|
+        next if !system_ids.include?(pool.pump_id)
+
+        watts += 1491
```

Comment:
@lixiliu - this has all pool pumps receiving the 2 HP pool pump capacity. Open question if we want to retag / update the actual pool pumps in ResStock to get the labeling and energy consumption to match.

Reply:
I think we'd need to make some changes/updates on OS-HPXML's end since it appears the BuildResidentialHPXML measure _always_ writes an "unknown" pool pump type: https://github.com/NREL/OpenStudio-HPXML/blob/master/BuildResidentialHPXML/measure.rb#L7345.

Reply:
Same for spa pumps.

Reply:
Oh that's interesting that unknown type is always assigned.. I think there's two intertwined concepts here of power rating (and the associated panel capacity implications) and energy consumption. In `options_lookup.tsv` ResStock has 3 different pool pumps of different power ratings (0.75 HP, 1 HP, and National Average). For all of these `pool_pump_annual_kwh` is set to `auto`, but the `pool_pump_usage_multiplier` varies (0.75, 1.0, and 0.075 respectively).

The issue with this is that Lixi has identified that all of our pool pumps in ResStock are undersized. For the purposes of the panels project, she has suggested the capacity default be based on a 2 HP pool pump, which is likely more common in practice. I think this makes sense for this project, and I think it's fine to assign as a default in OS-HPXML. I don't think this is significant to call out as an argument in `options_lookup.tsv`. 

The related issue I'm highlighting is that ResStock seems to have undersized pool pumps, which leads to both an energy issue as well as a labeling discrepancy (i.e., ResStock says it's a 1 HP pool pump, but OS-HPXML assumes the panel capacity is equivalent to a 2 HP pool pump).

@lixiliu - I'd propose we leave everything as-is and open a `resstock-estimation` issue on improving pool pump distributions, highlighting the capacity discrepancy. Wanted your signoff on this though